makepkg.sh

```diff
@@ -2248,16 +2248,74 @@ run_fpm() {
+#
+# fpm - https://github.com/jordansissel/fpm/wiki
+#
+# --post-install   FILE (DEPRECATED, use --after-install) A script to be run after package installation
+# --pre-install    FILE (DEPRECATED, use --before-install) A script to be run before package installation
+# --post-uninstall FILE (DEPRECATED, use --after-remove) A script to be run after package removal
+# --pre-uninstall  FILE (DEPRECATED, use --before-remove) A script to be run before package removal
+# --after-install  FILE A script to be run after package installation
+# --before-install FILE A script to be run before package installation
+# --after-remove   FILE A script to be run after package removal
+# --before-remove  FILE A script to be run before package removal
+# --after-upgrade  FILE A script to be run after package upgrade. If not specified,  --before-install, --after-install, --before-remove, and --after-remove will behave in a backwards-compatible manner (they will not be upgrade-case aware).  Currently only supports deb and rpm packages.
+# --before-upgrade FILE A script to be run before package upgrade. If not specified, --before-install, --after-install, --before-remove, and --after-remove will behave in a backwards-compatible manner (they will not be upgrade-case aware). Currently only supports deb and rpm packages.
+#
+# rpm - https://fedoraproject.org/wiki/Packaging:Scriptlets
+#
+# %pretrans	install	upgrade
+# %pre 		install	upgrade
+# %post 	install	upgrade
+# %preun 		upgrade	uninstall
+# %postun 		upgrade	uninstall
+# %posttrans 	install	upgrade
+
+	if [[ "$pre_install" = "" ]]; then
+		local before_install_param=''
+	else
+		local before_install_param="--before-install \"${startdir}/${pre_install}\""
+	fi
+	if [[ "$post_install" = "" ]]; then
 		local after_install_param=''
 	else
-		local after_install_param="--after-install \"${startdir}/${install}\""
+		local after_install_param="--after-install \"${startdir}/${post_install}\""
+        fi
+	if [[ "$pre_upgrade" = "" ]]; then
+		local before_upgrade_param=''
+	else
+		local before_upgrade_param="--before-upgrade \"${startdir}/${pre_upgrade}\""
 	fi
+	if [[ "$post_upgrade" = "" ]]; then
+		local after_upgrade_param=''
+	else
+		local after_upgrade_param="--after-upgrade \"${startdir}/${post_upgrade}\""
+	fi
+	if [[ "$pre_remove" = "" ]]; then
+		local before_remove_param=''
+	else
+		local before_remove_param="--post-remove \"${startdir}/${pre_remove}\""
+	fi
+	if [[ "$post_remove" = "" ]]; then
+		local after_remove_param=''
+	else
+		local after_remove_param="--after-remove \"${startdir}/${post_remove}\""
+	fi
+
 	local nm=$1; shift
 	local cmd="fpm -s dir -t rpm --force -a $fpm_arch"
 	cmd="$cmd $rpm_dist_param"
 	cmd="$cmd --rpm-os linux"
-	# cmd="$cmd --debug-workspace"  # skips cleanup of /tmp and allows inspection of SPECFILE
+	cmd="$cmd --debug-workspace"  # skips cleanup of /tmp and allows inspection of SPECFILE
```

Comment:
No, it shouldn't.  The tarball that is created includes all of that stuff, so there is really no reason to leave it in `/tmp`.  I'll amend the PR with the change.